src/main/java/org/mskcc/cmo/messaging/impl/JSGatewayImpl.java

```diff
@@ -150,15 +150,14 @@ public void subscribe(String subject, Class messageClass,
             ConsumerConfiguration consumerConfig = ConsumerConfiguration.builder()
                     .durable(consumerName)
                     .filterSubject(filterSubject)
-                    .deliverPolicy(DeliverPolicy.New)
                     .ackPolicy(AckPolicy.All)
-                    .replayPolicy(ReplayPolicy.Instant)
+                    .deliverPolicy(DeliverPolicy.New)
```

Comment:
Can you comment on why the replay policy needed to be removed?

We are setting up consumers in the init script with `--replay=instant` so I would want to make sure that either 1) default is going to be instant anyway and removing it from here doesn't matter and 2) that the consumers will still be linked up correctly and we won't have to deal with that duplicate consumer issue we had before

Reply:
We can actually remove all the policies since they're redundant. In this case instant reply is a default setting